src/test/scala/io/dockstore/DockstoreWebUser.scala

```diff
@@ -60,8 +70,31 @@ class DockstoreWebUser extends Simulation {
       SearchPage.search,
       TRS.searchAndDrillDown,
       Metadata.go
-    )
+    ),
+
+    scenario(loggedInUserFlow_1_9).feed(tokenFeeder).exec(
+      LoggedInHomepage.loggedInHomepage,
+      Accounts.accountsPage,
+      io.dockstore.release_1_9.SearchPage.search,
```

Comment:
Would like nicer if you imported the package and didn't need the full package name here. OTOH, your package name indicates it's 1.9. Hmm...

Reply:
the whole package gets imported because `SearchPage` is already in use so has to distinguish between the two. Just going to keep this way.